numba/cuda/tests/cudadrv/test_linker.py

```diff
@@ -269,6 +284,29 @@ def test_get_shared_mem_per_specialized(self):
         shared_mem_size = compiled_specialized.get_shared_mem_per_block()
         self.assertEqual(shared_mem_size, 800)
 
+    def test_get_no_local_memory(self):
```

Comment:
I still don't think this test is safe, as the compiler isn't guaranteed to allocate zero local memory even for an empty kernel - similar to the analagous test in https://github.com/numba/numba/pull/8295, I think this should just be removed.

Reply:
@gmarkall test removed.